### PR TITLE
Changes for the Updated fields in the XSD for the November release

### PR DESCRIPTION
**Pull request by sharathkatteri**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,19 +2,19 @@ language:
     csharp
 
 install:
-    - sudo apt-get install nunit-console
+#    - sudo apt-get install nunit-console
```
Comment (gnongsie) [inline .travis.yml]: Why is this commented? This will cause the Travis CI check to fail.

File: .travis.yml [hunk 2]
```diff
@@ -2,19 +2,19 @@ language:
...
 
 before_script:
     - git submodule update --remote --recursive
   
 script:
-    - xbuild ./Authorize.NET/AuthorizeNET.csproj
-    - xbuild ./AuthorizeNETtest/AuthorizeNETtest.csproj
+    - msbuild ./Authorize.NET/AuthorizeNET.csproj
+    - msbuild ./AuthorizeNETtest/AuthorizeNETtest.csproj
```
Comment (gnongsie) [inline .travis.yml]: Why is this changed from xbuild to msbuild?

File: .travis.yml [hunk 3]
```diff
@@ -2,19 +2,19 @@ language:
...
 #   make mono happy by copying the config file with project name
     - cp AuthorizeNETtest/App.config AuthorizeNETtest/AuthorizeNETtest.config
     - nunit-console ./AuthorizeNETtest/AuthorizeNETtest.csproj -run=AuthorizeNet.Api.Controllers.MockTest -exclude Integration,NotWorkingOnMono
 
     - cp ./Authorize.NET/bin/Debug/AuthorizeNet.dll ./sample-code-csharp/
-    - xbuild ./sample-code-csharp/SampleCode.csproj
-    - xbuild ./sample-code-csharp/SampleCodeTest/SampleCodeTest.csproj 
+    - msbuild ./sample-code-csharp/SampleCode.csproj
+    - msbuild ./sample-code-csharp/SampleCodeTest/SampleCodeTest.csproj
```
Comment (gnongsie) [inline .travis.yml]: Why is this changed from xbuild to msbuild?

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs [hunk 4]
```diff
@@ -563,6 +597,7 @@ public static void creditCardMaskedType(creditCardMaskedType argument)
             {
                 cardArt(argument.cardArt);
                 if(argument.isPaymentToken) { argument.isPaymentTokenSpecified=true;}
+               // if (argument.isPaymentToken) { argument.isPaymentTokenSpecified = true; }
```
Comment (gnongsie) [inline Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs]: Remove commented code.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs [hunk 5]
```diff
@@ -787,7 +842,13 @@ public static void customerProfileType(customerProfileType argument)
                 if(0 <= argument.profileType) { argument.profileTypeSpecified=true;}
             }
         }
-		public static void securePaymentContainerErrorType(securePaymentContainerErrorType argument) 
+        public static void ContactDetailType(ContactDetailType argument)
+        {
+            if (null != argument)
+            {
+            }
```
Comment (gnongsie) [inline Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs]: This is redundant. Add some code here or remove the if block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs [hunk 6]
```diff
@@ -1597,6 +1660,19 @@ public static void updateMerchantDetailsResponse(updateMerchantDetailsResponse a
             {
             }
         }
+        public static void getCustomerPaymentProfileNonceRequest(getCustomerPaymentProfileNonceRequest argument)
+        {
+            if (null != argument)
+            {
+            }
+        }
```
Comment (gnongsie) [inline Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs]: This is redundant. Add some code here or remove the if block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org [hunk 7]
```diff
@@ -814,6 +866,12 @@ transRetailInfoType() {
             {
             }
         }
+        public static void ContactDetailType(ContactDetailType argument) 
+        {
+            if(null != argument) 
+            {
+            }
```
Comment (gnongsie) [inline Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org]: This is redundant. Add some code here or remove the if block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org [hunk 8]
```diff
@@ -1548,6 +1608,19 @@ System.Nullable<
             {
             }
         }
+        public static void getCustomerPaymentProfileNonceRequest(getCustomerPaymentProfileNonceRequest argument) 
+        {
+            if(null != argument) 
+            {
+            }
+        }
```
Comment (gnongsie) [inline Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org]: This is redundant. Add some code here or remove the if block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs (hunk 5 above)
Comment (sharathkatteri) [reply Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs]: Removed the IF block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs (hunk 6 above)
Comment (sharathkatteri) [reply Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.cs]: Removed the IF block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org (hunk 7 above)
Comment (sharathkatteri) [reply Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org]: Removed the IF block.

File: Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org (hunk 8 above)
Comment (sharathkatteri) [reply Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org]: Removed the IF block.

File: .travis.yml (hunk 1 above)
Comment (sharathkatteri) [reply .travis.yml]: Made changes
